cos_alerter/daemon.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+import datetime
+import durationpy
+import json
+import signal
+import subprocess
+import sys
+import textwrap
+import time
+
+from .alerter import DataWriter, config, send_notifications
+
+
+def notify(time_data):
+    if not time.time() - time_data.notify_time > durationpy.from_str(config['notify']['repeat_interval']).total_seconds():
+        #TODO log something here
+        return
+    time_data.notify_time = time.time()
+    last_alert_time = datetime.datetime.fromtimestamp(time_data.alert_time).strftime('%A, %B %d %Y %I:%M%p %Z')
+    title = '**Alertmanager is Down!**'
+    body = textwrap.dedent(f'''
+    Your Alertmanager instance seems to be down!
```

Comment:
No indentation Python hurts.

```python
body = "Your Alertmanager instance seems to be down! \n "+
             f"It has not alerted COS-alerter since {last_alert_time}."
```
?

Reply:
Agreed. Fixed.